.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Erlang/OTP ${{matrix.otp}}
+    strategy:
+      matrix:
+        otp: ['24.3', '25.3']
+        rebar3: ['3.20']
```

Comment:
Should we expand the matrix to cover a greater spectrum of OTP and rebar3 versions?